apps/charterafrica/src/pages/api/v1/external/[...service].page.js

```diff
@@ -0,0 +1,35 @@
+import fetch from "node-fetch";
```

Comment:
Why external?

1. I'm assuming this is service specific i.e. YouTube in this PR.
2. We can probably keep navigation structure, for example, `opportunities/consultations/youtube` or videos or multimedia, etc.